.env.sample

```diff
@@ -22,6 +22,11 @@ APPLE_PRIVATE_KEY=your-apple-private-key
 APPLE_CLIENT_ID=your-apple-client-id
 APPLE_AUTH_REDIRECT_URI=http://127.0.0.1:8000/auth/apple/callback
 
+MICROSOFT_CLIENT_ID=client-id
+MICROSOFT_CLIENT_SECRET=client-secret
```

Comment:
question: should we use a certificate?

Reply:
Made an issue #321 to address this. It wouldn't be a lot of new code, but it might be easier to do it in a separate PR. What do you think @paulschreiber ? Would you prefer just to do it in this PR?

Reply:
Separate PR is fine.